rust/arrow/src/datatypes/native.rs

```diff
@@ -50,6 +50,12 @@ pub trait ArrowNativeType:
         None
     }
 
+    /// Convert native type to isize.
```

Comment:
I think this is a good pattern and follows the existing `to_usize` etc that convert the arrow types to rust types. 👍